docs/site/src/demos/layout/Interactive.js

```diff
@@ -0,0 +1,180 @@
+                aria-label="padding"
+              />
+            </Block>
+            <Block layout="row" align="center">
+              <Text type="subheading">Margin</Text>
+              <Switch
+                checked={margin}
+                onChange={this.handleChange('margin')}
+                aria-label="margin"
+              />
+            </Block>
+            <Block layout="row" align="center">
+              <Text type="subheading">Fill</Text>
+              <Switch
+                checked={fill}
+                onChange={this.handleChange('fill')}
+                aria-label="fill"
+              />
+            </Block>
+            <Block layout="row" align="center">
+              <Text type="subheading">Wrap</Text>
+              <Switch
+                checked={wrap}
+                onChange={this.handleChange('wrap')}
+                aria-label="wrap"
+              />
+            </Block>
+          </Block>
+          <Block flex>
+            <Text type="title">Justification</Text>
+            <RadioGroup
+              aria-label="Justification"
+              selectedValue={justification}
+              onChange={this.handleChange('justification')}
+            >
+              <Radio label="Start" value="start" />
+              <Radio label="Center" value="center" />
+              <Radio label="End" value="end" />
+              <Radio label="Space Between" value="space-between" />
+              <Radio label="Space Around" value="space-around" />
+            </RadioGroup>
+          </Block>
+          <Block flex>
+            <Text type="title">Alignment</Text>
+            <RadioGroup
+              aria-label="Alignment"
+              selectedValue={alignment}
+              onChange={this.handleChange('alignment')}
+            >
+              <Radio label="Start" value="start" />
+              <Radio label="Center" value="center" />
+              <Radio label="End" value="end" />
+              <Radio label="Stretch" value="stretch" />
+            </RadioGroup>
+          </Block>
+        </Block>
+        <Button
+          fab
+          accent
+          disabled={cells.length >= 30}
+          onClick={this.setCell()}
+          className={add}
+        >
+          <span className="material-icons">add</span>
+        </Button>
+        <Block flex className={demo} layout="row">
+          <Block
+            scroll
+            className={zone}
+            flex
+            layout={direction}
+            justify={justification}
+            align={alignment}
+            padding={padding}
+            margin={margin}
+            fill={fill}
+            wrap={wrap}
+          >
+            {
+              cells.map((cell, i) => (
+                <Block {...cell} layout="row" align="center">
+                  <IconButton accent onClick={this.setCell(i)}>delete</IconButton>
+                  <Button>{`Cell ${i + 1}`}</Button>
```

Comment:
I added the button to make it a menu to set flex value to that block. I don't know if I am overdoing the demo.